simulation/sim/sim_utils.go

```diff
@@ -1,12 +1,16 @@
 package sim
```

Comment:
We should refrain from using `utils` file.
This file contains `NewSimApp`, so this is the main file of the package, could keep the same name as the package.

In the end I think the organization can be as following:
```
simulation/
 simulation.go
 simulation_test.go
 state.go
 config.go
```

Reply:
Made this change , however I think functions like `PrintStats` and `CheckExportSimulation` should go into a utils file under the same directory

Reply:
I don't see any issue having these in `simulation.go`, these are general functions

Reply:
okay, I moved it to simulations.go